asmcomp/compilenv.mli

```diff
@@ -14,7 +14,7 @@
 
 open Cmx_format
 
-val reset: ?packname:string -> string -> unit
+val reset: ?packname:string -> sourcefile:Timings.build_kind -> string -> unit
```

Comment:
To someone with no knowledge of the `Timings` module, having a `sourcefile` parameter with type `Timings.build_kind` is not very intuitive.